lib/EnsEMBL/REST/Model/ga4gh/Beacon.pm

```diff
@@ -368,20 +370,24 @@ sub get_beacon_allele_request {
     $beaconAlleleRequest->{$field} = $data->{$field};
   }
 
+  if (exists $data->{start}) {
+    $beaconAlleleRequest->{start} = $data->{start} + 0;
+  }
+
   if (exists $data->{end}) {
-    $beaconAlleleRequest->{end} = $data->{end};
+    $beaconAlleleRequest->{end} = $data->{end} + 0;
   }
   if (exists $data->{startMin}) {
-    $beaconAlleleRequest->{startMin} = $data->{startMin};
+    $beaconAlleleRequest->{startMin} = $data->{startMin} + 0;
   }
   if (exists $data->{startMax}) {
-    $beaconAlleleRequest->{startMax} = $data->{startMax};
+    $beaconAlleleRequest->{startMax} = $data->{startMax} + 0;
   }
   if (exists $data->{endMin}) {
-    $beaconAlleleRequest->{endMin} = $data->{endMin};
+    $beaconAlleleRequest->{endMin} = $data->{endMin} + 0;
   }
   if (exists $data->{endMax}) {
-    $beaconAlleleRequest->{endMax} = $data->{endMax};
+    $beaconAlleleRequest->{endMax} = $data->{endMax} + 0;
```

Comment:
why do you +0 here?

perhaps $data->{whatever} can be undef?
in that case would it be better to  // 0 ?